chains/apechain-testnet.json

```diff
@@ -0,0 +1,23 @@
+{
+  "alias": "apechain-testnet",
+  "decimals": 18,
+  "explorer": {
+    "api": {
+      "key": {
+        "required": false
+      },
+      "url": "https://jenkins.explorer.caldera.xyz/api"
+    },
+    "browserUrl": "https://jenkins.explorer.caldera.xyz/"
+  },
+  "id": "1798",
+  "name": "ApeChain testnet",
```

Comment:
This testnet is a L3 of arbitrum-sepolia-testnet, so I'll update the alias of this one to be apechain-arbitrum-sepolia-testnet. The mainnet will be L2 of arbitrum, so it will be called apechain-arbitrum.